src/content/app/entity-viewer/variant-view/variant-view-sidebar/overview/MainAccordion.tsx

```diff
@@ -0,0 +1,141 @@
+    {
+      [styles.entityViewerAccordionButtonDisabled]: true
+    }
+  );
+
+  return (
+    <div className={styles.accordionContainer}>
+      <Accordion
+        className={styles.entityViewerAccordion}
+        allowMultipleExpanded={true}
+      >
+        <AccordionItem
+          className={styles.entityViewerAccordionItem}
+          uuid={'function'}
+        >
+          <AccordionItemHeading className={styles.entityViewerAccordionHeader}>
+            <AccordionItemButton className={styles.entityViewerAccordionButton}>
+              Alleles
+            </AccordionItemButton>
+          </AccordionItemHeading>
+          <AccordionItemPanel
+            className={styles.entityViewerAccordionItemContent}
+          >
+            <Alleles alleles={variant.alleles} />
+          </AccordionItemPanel>
+        </AccordionItem>
+
+        <AccordionItem
+          className={styles.entityViewerAccordionItem}
+          uuid={'other_data_sets'}
+        >
+          <AccordionItemHeading className={styles.entityViewerAccordionHeader}>
+            <AccordionItemButton
+              className={disabledAccordionButtonClass}
+              disabled={true}
+            >
+              In this region
+            </AccordionItemButton>
+          </AccordionItemHeading>
+          <AccordionItemPanel
+            className={styles.entityViewerAccordionItemContent}
+          >
+            <div>No data available</div>
+          </AccordionItemPanel>
+        </AccordionItem>
+
+        <AccordionItem
+          className={styles.entityViewerAccordionItem}
+          uuid={'other_data_sets'}
+        >
+          <AccordionItemHeading className={styles.entityViewerAccordionHeader}>
+            <AccordionItemButton
+              className={disabledAccordionButtonClass}
+              disabled={true}
+            >
+              Synonyms
+            </AccordionItemButton>
+          </AccordionItemHeading>
+          <AccordionItemPanel
+            className={styles.entityViewerAccordionItemContent}
+          >
+            <div>No data available</div>
+          </AccordionItemPanel>
+        </AccordionItem>
+      </Accordion>
+    </div>
+  );
+};
+
+type AlleleProps = {
+  alleles: {
+    allele_sequence: string;
+    reference_sequence: string;
+    allele_type: {
+      value: string;
+    };
+  }[];
+};
+const Alleles = (props: AlleleProps) => {
+  const { referenceAllele, alternativeAlleles } = getReferenceAndAltAlleles(
+    props.alleles
+  );
+
+  return (
+    <div>
+      <div className={styles.row}>
+        <div className={styles.label}>Ref</div>
+        <div className={styles.value}>
+          {referenceAllele?.reference_sequence}
+        </div>
+      </div>
+      <div className={styles.row}>
+        <div className={styles.label}>Alt</div>
+        <div className={styles.value}>
+          {alternativeAlleles.map((allele, index) => (
+            <div key={index}>{allele.allele_sequence}</div>
```

Comment:
Could you ask Andrea how to deal with long alts?

For example, try `variant:21:45988497:rs4647797`.

_(Not a blocker for this PR; can be addressed later)_

Reply:
Added 18ch but it doesn't really take exact char length. How to solve this?

Reply:
Change the string in javascript :-)